packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -548,6 +578,8 @@ class _RangeSliderState extends State<RangeSlider> with TickerProviderStateMixin
     enableController.dispose();
     startPositionController.dispose();
     endPositionController.dispose();
+    startFocusNode.dispose();
```

Comment:
This can be passed-in focus node, in that case, we shouldn't dispose it

Reply:
removed passed-in focus node